rs/nns/governance/src/neuron/types.rs

```diff
@@ -348,14 +353,57 @@ impl Neuron {
             > 0
     }
 
+    fn deciding_voting_power_adjustment_factor(
+        duration_since_voting_power_refreshed: Duration,
+    ) -> Decimal {
+        let linear_map = LinearMap::new(
+            Decimal::from(6 * ONE_MONTH_SECONDS)..Decimal::from(7 * ONE_MONTH_SECONDS), // from
+            Decimal::from(1)..Decimal::from(0),                                         // to
+        );
+
+        linear_map
+            .apply(Decimal::from(
+                duration_since_voting_power_refreshed.as_secs(),
+            ))
+            .clamp(Decimal::from(0), Decimal::from(1))
+    }
+
+    /// How much swap this neuron has when it casts its vote on proposals.
```

Comment:
Not clear to me what "swap" means in this context.

Reply:
I think what I meant to say was "sway".